app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharecterView.kt

```diff
@@ -28,26 +38,26 @@ class CharecterView(activity: MainActivity) {
     fun showToastNoItemToShow() {
         val activity = activityRef.get()
         if (activity != null) {
-            val message = activity.baseContext.resources.getString(R.string.message_no_items_to_show)
+            val message = activity.baseContext.resources.getString(com.puzzlebench.clean_marvel_kotlin.R.string.message_no_items_to_show)
```

Comment:
It is necessary all the path com.puzzlebench.clean_marvel_kotlin.

Reply:
Done!